app/presenters/tree_node_builder.rb

```diff
@@ -112,7 +112,9 @@ def build
     when MiqAlertSet          then generic_node(object.description, "miq_alert_profile.png")
     when MiqReport            then generic_node(object.name, "report.png")
     when MiqReportResult      then miq_report_node(format_timezone(object.last_run_on, Time.zone, 'gtl'),
-                                                   get_rr_status_image(object), object.name, object.status.downcase)
+                                                   get_rr_status_image(object),
+                                                   object.name,
+                                                   (object.try(:task_status) || object.status).downcase)
```

Comment:
...and also because task_status doesn't appear in the model anywhere, someone might come along in the future and see this and just remove it going "well, there is no task_status in the model, so this can't even happen".